### DISCUSSION TITLE
Show Popup to Client

### DISCUSSION THREAD

**Discussion by darouca** [amidaware/tacticalrmm#1987] [General]
Hi.

Is it possible for the RMM to send a message to the client to accept or not remote access?

**Comment by dinger1986** [comment]
https://docs.tacticalrmm.com/tipsntricks/#enable-remote-control-options

**Reply by darouca** [reply]
Hi.

Tks.

**Comment by Fabsaint** [comment]
Is possible to have this feature by users or group policy  ?

**Reply by Fabsaint** [reply]
Nobody answer ...

**Reply by dinger1986** [reply]
no, its globally set

**Reply by P6g9YHK6** [reply]
you can acctualy manage this in mesh itself for fine tuning per device group/users

**Reply by dinger1986** [reply]
you can but if users are synced again then it might over ride it

**Comment by Fabsaint** [comment]
ok thanks...but i can't use new groups created in mesh on rmm right ?

**Reply by P6g9YHK6** [reply]
if i remember correctly (i could be wrong)
you can move the devices to whatever group you want the sync does not care about where the device is.

**Reply by dinger1986** [reply]
That is correct yes, new machines will go into the tacticalrmm group.
